contracts/src/v0.8/interfaces/VRFCoordinatorV2Interface.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface VRFCoordinatorV2Interface {
```

Comment:
no getter function to "view" a subscription?

Reply:
added, actually there were a couple other sub related functions we should expose here too